src/plugins/data/public/query/filter_manager/lib/generate_filter.test.ts

```diff
@@ -219,8 +246,34 @@ describe('Generate filters', () => {
     expect(filter.query.range).toEqual({
       [FIELD.name]: {
         format: 'date_time',
-        gte: expect.stringContaining('2022-08-01T00:00:00'),
-        lte: expect.stringContaining('2022-08-01T00:00:00'),
+        gte: expect.stringContaining('2022-08-01'),
+        lte: expect.stringContaining('2022-08-01'),
+      },
+    });
+  });
+
+  it('should genereate a range filter when date_nanos type field is provided', () => {
```

Comment:
small typo, sounds like a mixture of generate and create ... which could be future word 
```suggestion
  it('should generate a range filter when date_nanos type field is provided', () => {
```